src/components/views/rooms/RoomTile.tsx

```diff
@@ -453,6 +464,16 @@ export default class RoomTile extends React.PureComponent<IProps, IState> {
             const isLowPriority = roomTags.includes(DefaultTagID.LowPriority);
             const lowPriorityLabel = _t("Low Priority");
 
+            const inRoom = this.props.room.getMyMembership() === "join";
+            const userId = MatrixClientPeg.get().getUserId();
+            let canInvite = inRoom;
+            const powerLevels = this.props.room.currentState
+                .getStateEvents("m.room.power_levels", "")
+                ?.getContent();
+            const me = this.props.room.getMember(userId);
+            if (powerLevels && me && powerLevels.invite > me.powerLevel) {
+                canInvite = false;
+            }
```

Comment:
This got stuck in review so long that I ended up adding a `canInvite()` method on a room :D
Could we use it here pelase

Reply:
Sure